components/GameResult/GameResult.tsx

```diff
@@ -0,0 +1,76 @@
+'use client';
+import Image from 'next/image';
+import classNames from 'classnames';
+import styles from './GameResult.module.scss';
+import imgWin from 'public/images/win.png';
+import playerImg from 'public/logo/account-avatar.png';
+import playerImg2 from 'public/logo/account-avatar2.png';
```

Comment:
We don't use our own player avatars, everything is provided by rainbowkit.

Reply:
it's a temporary solution for the layout